Communication/src/com/kubasz561/roulette/common/JSONMessage.java

```diff
@@ -1,12 +1,104 @@
 package com.kubasz561.roulette.common;
 
+import java.util.HashMap;
+import java.util.Map;
+
+
 //TODO: Opakować ładnie obiektowo jsona i metodki do niego do jakiegoś parsowania z raw_stringa do słownika Javowego
 public class JSONMessage{
     public String rawJSONString;
     public MessageType msgType;
+    private Map<String, String> dictionary = new HashMap<>();
     public JSONMessage(String rawMessage, MessageType type)
     {
         rawJSONString = rawMessage;
         msgType = type;
+        dictionaryCreate();
+    }
+
+    private void dictionaryCreate(){
```

Comment:
Kuba czaje że się naklepałeś , ale tu masz to zrobione o wiele prościej 
https://stackoverflow.com/questions/6737022/parse-json-string-to-dictionarystring-integer-with-gson